tests/test_wave_01.py

```diff
@@ -139,14 +147,19 @@ def test_moves_movie_from_watchlist_to_watched():
     updated_data = watch_movie(janes_data, movie_to_watch["title"])
 
     # Assert
-    assert len(updated_data["watchlist"]) is 1
-    assert len(updated_data["watched"]) is 2
+    assert len(updated_data["watchlist"]) == 1
+    assert len(updated_data["watched"]) == 2
     
     # *******************************************************************************************
     # ****** Add assertions here to test that the correct movie was added to "watched" **********
     # *******************************************************************************************
 
-@pytest.mark.skip()
+
+    assert updated_data["watched"][1]["title"] == MOVIE_TITLE_1
+    assert updated_data["watched"][1]["genre"] == GENRE_1
+    assert updated_data["watched"][1]["rating"] == RATING_1
```

Comment:
In this case I suggest `assert HORROR_1 in updated_data["watched"]`.  The requirements don't specify in what order the the movie should be added to the "watched" list.